QUEST_JOBSTEP.tsv

```diff
@@ -2,22 +2,22 @@ QUEST_JOBSTEP_20150317_000001	Bokor. You’ve chosen a difficult path. {nl}Refer
 QUEST_JOBSTEP_20150317_000002	You can only get it by yourself. {nl}Check your Adventure Journal to see how you can make the doll. 
 QUEST_JOBSTEP_20150317_000003	Congratulations, a new Bokor is born. {nl}When I first chose the path of a Bokor, I started by creating a doll of my own. 
 QUEST_JOBSTEP_20150317_000004	Peltasta Master
-QUEST_JOBSTEP_20150317_000005	If you want to receive my teachings then you have to prove your worth first.{nl}Here’s a challenge. Defeat some Vubbes on the 1st Floor of the Crystal Mine.
-QUEST_JOBSTEP_20150317_000006	How can you bear the name of Peltasta if you can't even handle that much? 
+QUEST_JOBSTEP_20150317_000005	$If you want to receive my teachings then you have to prove your worth first.{nl}Here’s a challenge: Defeat some Vubbes on the 1st Floor of the Crystal Mine.
+QUEST_JOBSTEP_20150317_000006	$How can you bear the title of Peltasta if you can't even handle that much? 
 QUEST_JOBSTEP_20150317_000007	Good. {nl}I hope your new growth leads to you saving the world. 
 QUEST_JOBSTEP_20150317_000008	Highlander Master
 QUEST_JOBSTEP_20150317_000009	I'm so thankful that you want to be the successor of my swordsmanship, but I suspect that you won't follow my teachings well.{nl}I will test you by having you defeat the Vubbes that are occupying the 1st floor of the Crystal Mine.
-QUEST_JOBSTEP_20150317_000010	Hurry. {nl} It's a simple challenge, but Vubbes are a threat to the Miner's Village.
+QUEST_JOBSTEP_20150317_000010	$Hurry. {nl} It's a simple challenge, but Vubbes are a threat to the Miners' Village.
 QUEST_JOBSTEP_20150317_000011	Good work, you've completed your initiation in becoming a Highlander. {nl} Keep up the hard work and dedication.
 QUEST_JOBSTEP_20150317_000012	Pyromancer Master
 QUEST_JOBSTEP_20150317_000013	I can see that you want to learn magic from me, but I can't teach just anyone. {nl}I will see how well you can assist my research and then decide. 
 QUEST_JOBSTEP_20150317_000014	Good work. Thanks to you I can start the next step of my research. {nl}I will teach you Pyromancer magic as promised. 
 QUEST_JOBSTEP_20150317_000015	Good, you're here. I need you to defeat the evil spirits that I will summon. {nl}I will record the unique wavelength that is transmitted in the process. 
-QUEST_JOBSTEP_20150317_000016	I am doing research on the 4th state of matter. {nl}It is impossible without the cooperation of the Bokor Master, who deals with the spirits. 
+QUEST_JOBSTEP_20150317_000016	$I am doing research on the 4th state of matter. {nl}It is impossible without the cooperation of the Bokor Master, who deals with spirits. 
```

Comment:
No, it's correct as is.  You could say "a research project", but saying "I'm doing research" is correct.